distribution/lib/Standard/Base/0.0.0-dev/src/Data/Set.enso

```diff
@@ -18,7 +18,9 @@ type Set
     ## PRIVATE
     Value (underlying_map : Map Any Nothing)
 
-    ## Constructs a new set from a vector.
+    ## PRIVATE
+       ADVANCED
+       Constructs a new set from a vector.
```

Comment:
Why are we making the Set hidden in CB? Isn't it useful to sometimes work with sets?